pathetic-model/src/main/java/org/patheloper/util/ChunkUtils.java

```diff
@@ -27,7 +33,16 @@ public long getChunkKey(final int x, final int z) {
      * Get the block type from a chunk snapshot at the given coordinates
      */
     public Material getMaterial(ChunkSnapshot snapshot, int x, int y, int z) {
-        return BLOCK_PARSER.getBlockMaterialAt(snapshot, x, y, z);
+        if (BukkitVersionUtil.isUnder(13)) {
+            if (materialMethod == null || blockTypeMethod == null) return null;
```

Comment:
you are double-checking here if the version is < 13

Reply:
I mean, yes. But at the same time I don't want to use the nullability check as a version check

Reply:
you could throw an illegalstate if one of them is null instead of returning null